mongodbatlas/resource_mongodbatlas_cloud_backup_schedule.go

```diff
@@ -455,3 +498,15 @@ func flattenPolicyItem(items []matlas.PolicyItem, frequencyType string) []map[st
 
 	return policyItems
 }
+
+func flattenExport(roles *matlas.CloudProviderSnapshotBackupPolicy) []map[string]interface{} {
+	exportList := make([]map[string]interface{}, 0)
+	test := matlas.CloudProviderSnapshotBackupPolicy{}
```

Comment:
[q] Why is this called `test`? It doesn’t seem to use for testing

Reply:
testing for null / empty struct

Reply:
renamed